src/operations/wedge.jl

```diff
@@ -23,25 +23,24 @@ export wedge, ∧
 # --- Operator aliases
 
 """
-    ∧(M::AbstractMultivector, N::AbstractMultivector)::AbstractMultivector
+    ∧(M, N)::AbstractMultivector
 
-Alias for the `wedge()` function. Compute the exterior product of `M` and `N`.
+Alias for the [`wedge()`](@ref wedge) function. Compute the exterior product of `M` and `N`.
 """
-∧(M::AbstractMultivector, N::AbstractMultivector) = wedge(M, N)
-
-∧(M::AbstractMultivector, x::Real) = wedge(M, x)
-∧(x::Real, M::AbstractMultivector) = wedge(x, M)
-
-∧(M::AbstractMultivector, v::Vector{<:Real}) = wedge(M, v)
-∧(v::Vector{<:Real}, M::AbstractMultivector) = wedge(v, M)
-
-∧(v::Vector{<:Real}, w::Vector{<:Real}) = wedge(v, w)
+∧(M, N) = wedge(M, N)
```

Comment:
Will this conflict with definitions of `∧` if another package defines it differently, as we're not restricting the types of its arguments anymore?

Reply:
Can you include this in the GitHub issues for how to deal with \cdot? I think it would be good to spend time thinking through a good strategy for dealing with these potential operator conflict issues.

For now, I think it's probably alright to leave the more general method definition, but you raise a good point.

If you're alright with this plan, can you "resolve" this conversation? (I have adjusted the repo settings to require that all conversations are resolved before pull requests can be merged). Thanks!

Reply:
I updated issue #40.